hardware_interface/include/hyperion.h

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/**
+ * @file    hyperion.c
+ * @author  Trung Tran
+ * @date    2020-06-04
```

Comment:
file name is wrong
I'm guessing that creation date is wrong if you're the author
also verify the other files to ensure these headers are right

Reply:
My bad. I just copy paste from other files. Is there an automatic way to do this?

Reply:
I don't know of any automatic system